### DISCUSSION TITLE
中英混杂 - Miao Yu | 于淼

### DISCUSSION THREAD

**Discussion by giscus** [yufree/yufree.cn#432] [General]
# 中英混杂 - Miao Yu | 于淼

https://yufree.cn/cn/2023/08/20/hybrid-lauguage/

**Comment by earfanfan** [comment]
>因此很多职业都可以解释为翻译，程序员翻译需求给机器、销售员翻译商品用途、科学家翻译自然法则、艺术家翻译情感、运动员翻译胜负欲……人的所有需求都需要翻译，有的还需要高度职业化翻译。

我最近听陈丹青的《局部》，里面讲了一点点绘画艺术发展的历史。在比较早期的时候，油画是以订件需求的形式存在的，宗教、皇族有需求，于是向画家下订单，那个时候颜料贵、不方便携带，画家们往往在画室内根据需求作画，既然有固定需求，那么画家作画便无法随意地表达情感。后来制作颜料的工艺提升了，单管颜料出现了，画家了可以更容易携带颜料去野外写生，会画画的人多了、学画的成本降低了，画作从订件需求的形式转换成自由买卖的形式，即画家根据自己的想法作画，随后委托中介卖出，画作开始成为画家表达情感的工具。再往后，用画可以表达的内容更丰富了，可以表达思想，表达新奇的概念、想象中的未来世界等等。

啰哩吧嗦写这么多，当然不是来找三水你吵架的。而是我发现之前总是吐槽你关注整体结构忽略局部细节这点其实是错了。人类创造出来的内容实在是太丰富了，以你这篇博客为例，你拟定了主题，自然是要围绕这个主题来思考的，倘若在“艺术家翻译情感”这里死扣细节，那这就太没完没了、无穷无尽了。正是因为繁杂、混沌处太多，丰富的细节只能做出取舍，取出支撑你论点的那一部分。所以说，我错了，以后再也不吐槽你这个了。

**Reply by yufree** [reply]
说到底还是“名可名，非常名”的问题，语言文字不见得能传递最精确的信息，跟对错没啥关系。

**Reply by earfanfan** [reply]
嗯，不过语言文字不能精确传递内容太正常了。

因为世间万物蕴含内容实在太丰富了，语言文字也不能用来表达所有。正是因为仅仅凭借语言文字还不够，人类创造了一大堆工具，比如最早出现的音乐、图画啊，近代出现的电影电视啊。然而，每每造出来一种工具，为了方便传递内容更精准，又同步造出来一套标准，同时约束着表达和解读两方面。这里借一下你平时的思路，随着时间推移，人类的需求越发复杂，各种工具又被造出来好几套标准，也就分出来许多不同派别和体系……

不造复杂的尽头会不会是回归简单，比如猴年马月三水你写博客的形式不再是产出文字，而是产生意念，俺这个读者下载下来可以直接塞脑子里解读。

话又说回来，语言文字这四个字里面，文字也是语言这个工具的一种解读标准啊，毕竟世界上存在不识字的人，但不识字也可以说话和听别人说话，就算是哑巴也可以用肢体语言来表达。日常生活中沟通起来不精确好像也没关系。比如阿木狗不会说话，我也看不大懂它全部的肢体语言，然而我跟阿木狗其实本来就不怎么需要沟通，一样把它养得很好。但是换到网络上面的文字交流，发出去的文字随时都可能会被误解，就算文字本身是传递内容的一套表达和解读标准，是否表达精确或解读精确也跟人心里的标准不统一有关。唉，我竟一时不知，到底是世界更复杂还是人心更复杂了。如果说是世界更复杂的话，地球上能产生那么多种语言文字最初确实就是因为有地理位置上的区格。如果说是人心更复杂的话，本菜鸟把自己绕进去了。回头会在福柯的知识考古学里再看到这些。

我记得列弛说言象意之间的关系自古以来是个神仙打架的问题，估计上面我絮叨的那么多你又可以一句话总结一下，哈哈。
